### ISSUE TITLE
Multipart parsing error (init): Multipart: Invalid boundary in C-T (malformed) - being received on a compliant header

### ISSUE THREAD

**Issue by nneul** [SpiderLabs/ModSecurity#1389]
Example header in the requests:

Content-Type: multipart/form-data; boundary="x-hx6COixr31cBVcPHyPgwef.gY5tAoL4RbHUpXrTPSUgO9K6IqTafzKBcmTJ.opw9lYit"

Content-Type: multipart/form-data; boundary="Sgt-3Wo5day44dJGZ/0KW/q0FRjE4hVqSxCGvkhnCwKAeuprMel1s0kZvXhe7/s5hrh/Pg"

According to the RFC2616 19.2.2 quotes are valid but optional. 

RFC2046 doesn't include it, but if you look over RFC2046 they show multiple examples with and without quotes in the doc itself:

```
     boundary := 0*69<bchars> bcharsnospace
     bchars := bcharsnospace / " "
     bcharsnospace := DIGIT / ALPHA / "'" / "(" / ")" /
                      "+" / "_" / "," / "-" / "." /
                      "/" / ":" / "=" / "?"
```

I'm going to file an issue with MS/Azure as well since I'm seeing this on their WAF, but the underlying ruleset is coming from this project.

**Comment by victorhora** [comment]
I'm closing this one as it's being already handled on owasp-modsecurity-crs and seems like it's already solved.

**Comment by davidjrh** [comment]
Hi, I'm getting the same issue with the latest released version 2.9.2, and the 3.0.2 rule set. I'm using the Windows IIS version.

Which mentioned rule is handling this?
